rllib/models/action_dist_v2.py

```diff
@@ -0,0 +1,119 @@
+from typing import Tuple
+import gym
+import abc
+
+from ray.rllib.utils.annotations import ExperimentalAPI
+from ray.rllib.utils.typing import TensorType, Union, ModelConfigDict
+
+
+@ExperimentalAPI
+class ActionDistributionV2(abc.ABC):
+    """The policy action distribution of an agent.
+
+    Args:
+        inputs: input vector to define the distribution over.
```

Comment:
Is this always a vector? Or could this also be a dict of tensors (multi-distribution) or a >1D tensor (multi-variate diag gaussian)?

Reply:
Oh this is a residue of the old stuff. Doesn't mean anything anymore. Removed. :)